src/compressed_tensors/offload/cache/disk.py

```
@@ -176,7 +176,11 @@ def _get_safe_open_device(device: "DeviceLikeType") -> str | int:
     :return: device argument to `safetensors.safe_open`
     """
     device = torch.device(device)
-    if device.type in ("cuda"):
+    if is_accelerator_type(device.type):
```

Ruff (0.15.9)

[error] Undefined name `is_accelerator_type`

(F821)